packages/lesswrong/components/posts/PostsEditForm.tsx

```diff
@@ -2,7 +2,6 @@ import React from 'react';
 import { Components, registerComponent, getFragment } from '../../lib/vulcan-lib';
 import { useSingle } from '../../lib/crud/withSingle';
 import { useMessages } from '../common/withMessages';
-import { Posts } from '../../lib/collections/posts';
```

Comment:
You can skip this file, just some small formatting changes from when I was using it